tests/code/test_GetAllFacilitators.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from app.logic.getAllFacilitators import getAllFacilitators
+
+
+@pytest.mark.integration
+def test_getAllFacilitators():
+    userFacilitator = getAllFacilitators()
+
+
+    assert userFacilitator[0].username == "ramsayb2"
+    assert userFacilitator[0].isFaculty == 0
+    assert userFacilitator[0].isFaculty == False

```

Comment:
Since you want it to be False, just check if it's False. the check against 0 doesn't really add anything